pkg/skaffold/runner/build_deploy.go

```diff
@@ -80,7 +80,7 @@ func (r *SkaffoldRunner) BuildAndTest(ctx context.Context, out io.Writer, artifa
 	}
 
 	if r.runCtx.SkipTests() {
-		logrus.Warnln("Skip tests flag (skip-tests) is set to true. Tests won't run.")
```

Comment:
Lets not print any message. Skaffold already prints a lot of output. The team is rework only printing what is relevant.

Reply:
Removed the log message.